docs/UserGuide.md

```diff
@@ -113,6 +113,8 @@ You're at work and you crave baked potatoes. Let's do another one:
 * Notice how `bake potatoes` is above `Learn Task Tracker`. That's because
 `bake potatoes` is of number 2 priority, while `Learn Task Tracker` with an
 undefined priority takes the default lowest priority number of 0.
+
+* If you create two floating tasks with the same PRIORITY, they will be shown in order of the time they were created.
```

Comment:
@potatowagon Actually it is based on 1. Priority, 2. Lexicographical order of name. As for insertion order... depends on whether the JavaFX `SortedList` we use actually uses a stable sort, but I think just to be safe maybe fudge this point a bit.